core/scss/components/lockup/_lockup.scss

```diff
@@ -26,26 +26,21 @@
   width: fit-content;
   text-decoration: none;
 
-  @media #{$breakpoint-md} {
+  @include grid-media('md') {
     @include grid-columns(auto auto 1fr);
```

Comment:
```suggestion
    @include grid-columns(auto auto 1fr, false);
```

Reply:
If we add "false" to the argument of grid-columns on line 23 also, it should fix the lockup funny business.  I don't know how to make a suggested change on a line that's not displayed on this page though.

Reply:
Thanks for this change. What do you think about defaulting to the equal columns ms fix? Do you  think we should reverse it and instead opt-in to an auto-ie fix?

Reply:
I'm fine with it defaulting to the equal column ms fix. Seems like equal column would be the simplest configuration and is common in a lot of layouts, so having it default to the simplest config makes sense to me.